packages/preview/not-jku-thesis/0.1.0/template/thesis.typ

```diff
@@ -0,0 +1,240 @@
+//#import "@local/not-jku-thesis:0.1.0": jku-thesis // for development
```

Comment:
It would be cleaner to completely remove this line for the version that you publish on Typst Universe. It may a bit confusing to users of your template to have this line at the top of their thesis when they create a new project.

Reply:
alright - I removed that line